pkg/didcomm/packer/jwe/authcrypt/unpack.go

```diff
@@ -22,22 +22,22 @@ import (
 // encrypted CEK.
 // The current recipient is the one with the sender's encrypted key that successfully
 // decrypts with recipientKeyPair.Priv Key.
-func (p *Packer) Unpack(envelope []byte) ([]byte, []byte, error) {
+func (p *Packer) Unpack(envelope []byte) ([]byte, []byte, []byte, error) {
 	jwe := &Envelope{}
 
 	err := json.Unmarshal(envelope, jwe)
 	if err != nil {
-		return nil, nil, fmt.Errorf("unpack: %w", err)
+		return nil, nil, nil, fmt.Errorf("unpack: %w", err)
 	}
 
 	recipientPubKey, recipient, err := p.findRecipient(jwe.Recipients)
 	if err != nil {
-		return nil, nil, fmt.Errorf("unpack: %w", err)
+		return nil, nil, nil, fmt.Errorf("unpack: %w", err)
```

Comment:
please, change the error message. The error above has the same message.

Reply:
changed the error above.